### PR TITLE
instructions for person 108

### LINKED ISSUE #108: Create "##Instructions for reviewer" for project Person
Please add a section to the bottom of this project's _index.md file.
The section heading should be EXACTLY "## Instructions for reviewer"

Basically we have to set reviewers up for success by telling them what success looks like and what to pay attention to when reviewing this project submission.

Description:

- [ ]  Look at a bunch of old reviews for this project for each cohort

- [ ]  Find the common feedback [This should help you get an idea of what instructions to create]

- [ ]  Write them out clearly

- [ ]  The section heading should be EXACTLY "## Instructions for reviewer"

### PR DESCRIPTION
**Pull request by kaleem99**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/oop/person/_index.md [hunk 1]
```diff
@@ -58,15 +58,24 @@ System.out.println(greeting)
...
 ```
-Hello, my name is Ryan and I am 30 years old. My interests are being a hardarse, agile and ssd hard drives.
+Hello, my name is Ryan and I am a 30 year old male. My interests are being a hardarse, agile and ssd hard drives.
 ```
 
 In OOP this is known as abstraction. We created a simple model of a more complex thing. We only represent the attributes and functionality that we need.
 
 When an object instance is created from a class, the class's constructor function is run to create it. This process of creating an object instance from a class is called instantiation — the object instance is instantiated from the class. `person` is an instance of `Person`
 
+## Instructions for reviewer
```
Comment (ry-oc) [inline content/projects/oop/person/_index.md]: General clean code should become a habit for both the recruits as well as all reviewers. This should not need to be mentioned in every project. Please remove the instructions relating to general clean code guidelines and only leave the project-specific instructions.

Also please move the review instructions to below the Acceptance criteria.

**Review by ry-oc — CHANGES_REQUESTED** [review]
Almost there, some changes but also, please complete the template provided when you open a PR:

Related issues: [please specify] - this def relates to a issue.

 I ran the linter and there were no errors - did you not manage to run the linter?

File: content/projects/oop/person/_index.md (hunk 1 above)
Comment (ry-oc) [reply content/projects/oop/person/_index.md]: @kaleem99 please double-check, it looks like general clean code guidelines are still being included in the project review instructions

File: content/projects/oop/person/_index.md [hunk 2]
```diff
@@ -69,4 +69,10 @@ When an object instance is created from a class, the class's constructor functio
...
+
+## Instructions for reviewer
+
+1. **Multiple Interests, Commas and Spacing** - Make sure that the interests array attribute can take in multiple interests and they all get printed out seperated with a comma and space, and the last interest should have an ``and`` before it also your class should contain and use all of the atributes above.
+2. **Single Interest or no Interests** - Check if the class can work with single interests, and the string output should be different from the multiple interest output instead of `my interests are` it should be `my interest is`, also if there are no interests then the output should a string with the first 3 attributes name, age, and male `Hello, my name is Ryan, my gender is male and I am 30 years old.` and you could add something like `I have know interests`
+3. **Competent or not yet competent** - If all of the instructions above have not been bet then the recruit should get a ``NYC`` with the instructions that have not been followed in his feedback.
```
Comment (ry-oc) [inline content/projects/oop/person/_index.md]: Point 3 seems to be a general point that applies to all projects? Please can you remove it.

**Review by ry-oc — CHANGES_REQUESTED** [review]

**Review by ry-oc — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Your capitalisation is a little strange. Treat headings like normal English, dont capitalise everything, capitalise the first word and proper nouns.

Eg:
**Multiple Interests, Commas and Spacing**
Should be: **Multiple interests, commas and spacing**

File: content/projects/oop/person/_index.md [hunk 3]
```diff
@@ -69,4 +69,9 @@ When an object instance is created from a class, the class's constructor functio
...
\ No newline at end of file
+Also, it's really imporant that you submit good clean code! MAke sure you name your variables clearly. Make sure you are following the convention of the langage you are using. Make sure your code in no way resembles spagetti.
+
+## Instructions for reviewer
+
+1. **Multiple interests, commas and spacing** - Make sure that the interests array attribute can take in multiple interests and they all get printed out separated with a comma and space, and the last interest should have an ``and`` before it also your class should contain and use all of the attributes above.
+2. **Single interest or no interests** - Check if the class can work with single interests, and the string output should be different from the multiple interest output instead of `my interests are` it should be `my interest is`, also if there are no interests then the output should a string with the first 3 attributes name, age, and male `Hello, my name is Ryan, my gender is male and I am 30 years old.` and you could add something like `I have know interests`.
```
Comment (sheenarbw) [inline content/projects/oop/person/_index.md]: To know something means to have knowledge of that thing. 

I have no interests means I have 0 interests.  'No' and 'know' mean very very different things.

**Review by sheenarbw — CHANGES_REQUESTED** [review]

**Review by ry-oc — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**